README.md

```diff
@@ -35,3 +39,156 @@ understand.
+These fake `mpirun` commands basically wrap around `mympirun` (which in turn wraps around the real `mpirun` command).
+
+Therefore, you should make sure that the location of the `mympirun` command and the fake `mpirun` is prepended to `$PATH`. If `mympirun` is provided via a module, you should load it *after* any other modules.
+
+
+# Detection of MPI and job scheduler
+
+`mympirun` detects which MPI implementation (OpenMPI, Intel MPI, ...) is being used, to figure out which `mpirun` command it should be using. This can be overridden using the `--setmpi` or `-M` option. An overview of known MPI implementations is available via `-m` or `--showmpi`.
+
+In addition, the job scheduler (e.g., Torque) is also detected automatically, and can be overridden via `--schedtype` or `-S`.
+If not job scheduler could be detected, local execution is assumed.
+An overview of known job scheduler is available via `-s` or `--showsched`.
+
+
+# Available resources
+
+`mympirun` will detect the available resources, and pass options to the `mpirun` command being used accordingly.
+
+By default, it will use all available cores, i.e.:
+
+* all cores on the *current* system, if the `local` scheduler is used
+* all cores assigned to the current job, if the `pbs` scheduler is used
+
+This can be changed if needed using the `--hybrid`, `--universe`, `--double` or `--multi` options, see below.
+
+Pinning of MPI processes to cores is also enabled by default (can be disabled using `--disable-pinmpi`).
+
+It will also leverage the Torque integration of the MPI library being used by default, by launching MPI processes using `pbsdsh` rather than `ssh`.
+
+
+# Configuring `mympirun`
+
+* `mympirun` command line options
+* `$MYMPIRUN_*` environment variables
+* configuration files (see `--configfiles`)
+
+# Controlling number of processes
+
+## `--hybrid` / `-h`
+The `hybrid` or `h` option requires one integer. This integer will be the number of processes started on each available physical node.
+        
+    $ echo $PBS_NUM_NODES
+    2
+
+    $ mympirun --hybrid 2 hostname
+    node2157.delcatty.os
+    node2157.delcatty.os
+    node2158.delcatty.os
+    node2158.delcatty.os
+
+## `--universe`
+The `universe`option also requires one integer. This integer will be the exact number of processes started by mympirun (independent of the number of nodes). When using `universe` for multiple processes, mympirun will try to balance the processes on the available nodes.
+        
+    $ echo $PBS_NUM_NODES
+    2
+
+    $ mympirun --universe 1 hostname
+    node2157.delcatty.os
+    
+
+## `--double` and `--multi`
+As the name suggests, when using the `double` option,  mympirun will start double the amount of processes as requested. The `multi`option works the same but it requires an integer, indicating the multiplier, for example, `multi 3` will start triple the amount of processes. This means `double` and `multi 2` will have the exact same effect.
```

Comment:
not 'as requested', but 'as it would without it'

also, use `--double` and `--multi` rather than `double` and `multi` (also do this for other options)

missing space in "`multi`option"